astester.imports.js

```diff
@@ -4,57 +4,155 @@ export function setExports(xpt) {
   exports = xpt;
 }
 
-export function local(memory) {
-  const Storage = new Map();
+function makeid(length) {
+  var result = '';
+  var characters =
+    'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  var charactersLength = characters.length;
+  for (var i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+function generateAddress() {
+  return 'A12' + makeid(47);
+}
+
+/* Ledger format :
+{
+    "Address" : {
+        "Storage" : {
+            "key1" : "value1",
+            "key2" : "value2"
+        },
+        "Contract" : "./pathOfTheAssemblyScriptContract",
+    }
+}
+*/
+export async function local(memory) {
+  const emptyAddress = {
+    storage: new Map(),
+    contract: '',
+  };
+  const defaultAddress = 'A12UBnqTHDQALpocVBnkPNy7y5CndUJQTLutaVDDFgMJcq5kQiKq';
+
+  const Ledger = new Map();
```

Comment:
why capital letter for instance of Map?

Reply:
> why capital letter for instance of Map?

For the `Ledger `one you mean ?

Reply:
yes

Reply:
To be honnest with you I don't know :')
I think as it's a `main` object I put a CAPS to highlight it .
But you right, for consistency we should keep the same format for all consts

I change it